### DISCUSSION TITLE
Misformatted group names

### DISCUSSION THREAD

**Discussion by grzesuav** [Aloxaf/fzf-tab#212] [Q&A] [answered]
Hi, I installed `fzf-tab`, and when I am in git repo and I click `TAB` I get

![image](https://user-images.githubusercontent.com/9568421/116458467-fb39d780-a864-11eb-82de-e8df4a31a8e6.png)

I think instead 

```shell
%F{yellow}-- remote head --%f
```
there should be group name within braces.

Any advice what can be wrong ?

**Comment by grzesuav** [comment, marked as answer]
Just found https://github.com/Aloxaf/fzf-tab/issues/24#issuecomment-574986122

**Reply by grzesuav** [reply]
I found the place where it is set, but the question is - how it should be fixed ? I mean - what value set there ?

**Reply by grzesuav** [reply]
Ok, the easiest way is to use 
```zsh
zstyle ':completion:*' format '[%d]'
```

which results in 
![image](https://user-images.githubusercontent.com/9568421/116461411-81a3e880-a868-11eb-9893-d93163eb6717.png)

**Reply by grzesuav** [reply]
@Aloxaf maybe worth adding this to README ? It took me a while to look onto issues and find that the pinned one is exaclty about this ?

**Reply by Aloxaf** [reply]
Thanks for your advice. I think I should find a way to remove them.
